tsconfig.json

```
@@ -1,15 +1,13 @@
 {
+    "extends": "@tsconfig/node12/tsconfig.json",
```

Looks like `tscpaths` is unable to resolve this.

```
Error: Cannot find module '/workspaces/eslint-remote-tester/@tsconfig/node12/tsconfig.json'
Require stack:
- /workspaces/eslint-remote-tester/node_modules/tscpaths/cjs/util.js
- /workspaces/eslint-remote-tester/node_modules/tscpaths/cjs/index.js
```

However relative path works:

```suggestion
    "extends": "./node_modules/@tsconfig/node12/tsconfig.json",
```